pkg/util/tests/runtime.go

```diff
@@ -0,0 +1,126 @@
+package tests
+
+import (
+	"net"
+	"os"
+	"path/filepath"
+
+	"github.com/rancher/k3s/pkg/daemons/config"
+	"github.com/rancher/k3s/pkg/daemons/control/deps"
+	"github.com/rancher/k3s/pkg/version"
+)
+
+// GenerateTestDataDir creates a temporary directory at "/tmp/k3s/<RANDOM_STRING>/".
+// The latest directory created with this function is soft linked to "/tmp/k3s/latest/".
+// This allows tests to replicate the "/var/lib/rancher/k3s" directory structure
+func GenerateTestDataDir(cnf *config.Control) error {
```

Comment:
Can you remove the "Test" portion from these funtion names? It's redudant as the package ref when used will provide context. ie `test.GenerateDataDir`.

Reply:
Removed